core/src/main/java/com/homeaway/streamingplatform/db/dao/impl/StreamDaoImpl.java

```diff
@@ -15,12 +15,7 @@
  */
 package com.homeaway.streamingplatform.db.dao.impl;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
+import java.util.*;
```

Comment:
* **no wildcard imports** except for unit-test helper/matchers please.

Reply:
Is there a Maven Plugin somewhere that disallows this?